cucumber/authenticators_gcp/features/step_definitions/authn_gcp_steps.rb

```diff
@@ -24,19 +24,68 @@
   set_annotation_to_resource("authn-gcp/#{annotation_name}", annotation_value)
 end
 
-# Sets all GCP annotations
+Given(/^I set "authn-gcp\/(service-account-id|service-account-email|project-id|instance-name)" annotation to function host "([^"]*)"$/) do |annotation_name, hostname|
+  i_have_a_resource "host", hostname
+
+  case annotation_name
+  when "service-account-id"
+    annotation_value = gcf_service_account_id
+  when "service-account-email"
+    annotation_value = gcf_service_account_email
+  when "project-id"
+    annotation_value = "_"
+  when "instance-name"
+    annotation_value = "_"
+  else
+    raise "Incorrect annotation name: '#{annotation_name}', expected: service-account-id|service-account-email|project-id|instance-name"
+  end
+
+  set_annotation_to_resource("authn-gcp/#{annotation_name}", annotation_value)
+end
+
+Given(/^I set "authn-gcp\/(service-account-id|service-account-email)" annotation with value: "([^"]*)" to host "([^"]*)"$/) do |annotation_name, annotation_value, hostname|
+  i_have_a_resource "host", hostname
+  set_annotation_to_resource("authn-gcp/#{annotation_name}", annotation_value)
+end
+
 Given(/^I set all valid GCP annotations to (user|host) "([^"]*)"$/) do |role_type, hostname|
   i_have_a_resource role_type, hostname
 
-  set_annotation_to_resource("authn-gcp/service-account-id", gcp_service_account_id)
-  set_annotation_to_resource("authn-gcp/service-account-email", gcp_service_account_email)
-  set_annotation_to_resource("authn-gcp/project-id", gcp_project_id)
-  set_annotation_to_resource("authn-gcp/instance-name", gcp_instance_name)
+  set_annotation_to_resource("authn-gcp/service-account-id", gce_service_account_id)
```

Comment:
its a little bit confusing:
Given **GCP** annotations
**gce_service_account** which relevant for gce/gcf
**project-id** which relevant for gce only
can we declare something like : gce_gcf_servce_account_id , gce_project_id or change to gcp_service_account_id so we will know its for both.